src/kvstore/raftex/RaftPart.cpp

```diff
@@ -1448,13 +1565,20 @@ void RaftPart::processAppendLogRequest(
     }
 
     // Reset the timeout timer
-    lastMsgRecvDur_.reset();
+    lastMsgRecvMs_ = time::WallClock::fastNowInMilliSec();
+    if (req.get_keep_alive()) {
+        resp.set_error_code(cpp2::ErrorCode::SUCCEEDED);
+        return;
+    }
 
     if (req.get_sending_snapshot() && status_ != Status::WAITING_SNAPSHOT) {
         LOG(INFO) << idStr_ << "Begin to wait for the snapshot"
                   << " " << req.get_committed_log_id();
         reset();
         status_ = Status::WAITING_SNAPSHOT;
+        resp.set_committed_log_id(committedLogId_);
```

Comment:
this line is under lock "raftLock_", why call "set_commited_log_id" again after line 1542?